Library/.rubocop.yml

```diff
@@ -355,15 +344,18 @@ Style/HashAsLastArrayItem:
     - "/**/Formula/**/*.rb"
     - "**/Formula/**/*.rb"
 
+Style/InverseMethods:
+  InverseMethods:
+    :blank?: :present?
+
 Style/InvertibleUnlessCondition:
   Enabled: true
   InverseMethods:
     # Favor `if a != b` over `unless a == b`
     :==: :!=
     # Unset this (prefer `unless a.zero?` over `if a.nonzero?`)
     :zero?:
-    # Don't require non-standard `exclude?` (for now at least) - it's not available in every file
-    :include?:
```

Comment:
This was introduced by the rubocop-rails config, and is no longer necessary to unset.